libraries/AP_GPS/AP_GPS.h

```diff
@@ -156,6 +156,47 @@ class AP_GPS
         GPS_OK_FIX_3D_RTK_FIXED = 6, ///< Receiving valid messages and 3D RTK Fixed
     };
 
+    /// GPS error bits. These are kept aligned with MAVLink by 
+    /// static_assert in AP_GPS.cpp
+    enum GPS_Errors {
```

Comment:
Can we make these enum-class and contained within the AP_GPS class?  Would allow for dropping the extra namespacing.